extensions/pdh/PerformanceDataMonitor.cpp

```diff
@@ -45,9 +45,13 @@ core::Property PerformanceDataMonitor::CustomPDHCounters(
 core::Property PerformanceDataMonitor::OutputFormatProperty(
     core::PropertyBuilder::createProperty("Output Format")->
     withDescription("Format of the created flowfiles")->
-    withAllowableValue<std::string>(JSON_FORMAT_STR)->
-    withAllowableValue(OPEN_TELEMETRY_FORMAT_STR)->
-    withDefaultValue(JSON_FORMAT_STR)->build());
+    withAllowableValues<std::string>({ PRETTY_JSON_FORMAT_STR, COMPACT_JSON_FORMAT_STR, PRETTY_OPEN_TELEMETRY_FORMAT_STR, COMPACT_OPEN_TELEMETRY_FORMAT_STR })->
+    withDefaultValue(PRETTY_JSON_FORMAT_STR)->build());
+
+core::Property PerformanceDataMonitor::DecimalPlaces(
+  core::PropertyBuilder::createProperty("Round to decimal places")->
+  withDescription("The number of decimal places to round the values to (blank for no rounding)")->
+  withDefaultValue("3")->build());
```

Comment:
If there is a default value, doesn't it mean it's impossible to have a blank value? Because it would just get replaced with the default.

Reply:
I was able to set empty string during testing with c2 protocoll, but changed it to blank nontheless thanks